packages/babel-preset-react-app/create.js

```diff
@@ -81,15 +89,13 @@ module.exports = function(api, opts, env) {
         {
           // We want Create React App to be IE 9 compatible until React itself
           // no longer works with IE 9
-          targets: {
-            ie: 9,
-          },
+          targets: isModern ? modernTargets : legacyTargets,
```

Comment:
You'll want to merge master in and resolve the conflicts. We will only want to set `targets` when using `modern` flag, otherwise it will use `browserslist`.

Reply:
so isModern && modernTargets so browserslist can override it

Reply:
Btw, big thanks for taking your time to review this, it's a work in progress and I'm super excited to have this in CRA

Reply:
We probably want to set targets to either `modernTargets` or `legacyTargets` when `modern` flag is set depending on which of the 2 builds is currently running. If `modern` flag isn't set then we won't define `targets` which will use `browserslist` instead. Does that make sense?

Reply:
so --modern is the code at hand, no flag is undefined because than we fall back to browserslist

Reply:
I think this is covered now